build.gradle

```diff
@@ -179,6 +189,21 @@ tasks.register("exportContainerData", DockerCopyFileFromContainer) {
 	remotePath = "/data/container-data"
 }
 
+tasks.register("configureWebserver", Copy) {
```

Comment:
Naming-wise, I think I'd like to keep the convention of `prepare*`, so something like `prepareWebserverConfig`. Thoughts?